src/components/Toast/ToastProvider.tsx

```diff
@@ -3,12 +3,16 @@ import React, { FC, useState } from 'react';
 import { LayoutAnimation } from 'react-native';
 import { ToastContext, ToastProps } from 'src/components/Toast';
 
+export interface ToastOptions extends ToastProps {
+  /** Duration in milliseconds */
+  duration?: number;
+}
 
 export const ToastProvider: FC = ({ children }) => {
-  const [toast, setToast] = useState<ToastProps>();
+  const [toast, setToast] = useState<ToastOptions>();
   const DEFAULT_DURATION = 5000;
 
-  const displayToast = (options: ToastProps) => {
+  const displayToast = (options: ToastOptions) => {
     if (!toast) {
       LayoutAnimation.configureNext(LayoutAnimation.Presets.easeInEaseOut);
       setToast(options);
```

Comment:
Has the issue with multiple subsequent toasts been resolved?

Reply:
Bump on this ^

Reply:
🤦‍♂️ not sure how I missed this.

What did we agree on here? Do we want a queue system where toasts appear right after eachother? Or maybe subsequent toasts to appear below each other? I can add this feature in another PR.

Reply:
Ahhh, yes. This has been fixed.

Reply:
Perfect 👍